test/RachisTests/DatabaseCluster/AtomicClusterReadWriteTests.cs

```diff
@@ -0,0 +1,281 @@
+            using var documentStore = GetDocumentStore(new Options { Server = leader, ReplicationFactor = nodes.Count });
+
+            var entity = new TestObj();
+            using (var source = GetDocumentStore())
+            {
+                using var session = source.OpenAsyncSession(new SessionOptions { TransactionMode = TransactionMode.ClusterWide });
+                await session.StoreAsync(entity);
+                await session.SaveChangesAsync();
+
+                var operation = await source.Smuggler.ExportAsync(new DatabaseSmugglerExportOptions(), documentStore.Smuggler);
+                await operation.WaitForCompletionAsync(TimeSpan.FromMinutes(1));
+            }
+
+            await LoadAndDeleteWhileUpdated(nodes, documentStore.Database, entity.Id);
+        }
+
+        private static async Task LoadAndDeleteWhileUpdated(List<RavenServer> nodes, string database, string entityId)
+        {
+            using var disposable = LocalGetDocumentStores(nodes, database, out var stores);
+
+            var amre = new AsyncManualResetEvent();
+            var amre2 = new AsyncManualResetEvent();
+            var task = Task.Run(async () =>
+            {
+                using var session = stores[0].OpenAsyncSession(new SessionOptions {TransactionMode = TransactionMode.ClusterWide});
+                var loaded = await session.LoadAsync<TestObj>(entityId);
+                amre.Set();
+
+                session.Delete(loaded);
+
+                await amre2.WaitAsync();
+                await Assert.ThrowsAnyAsync<ConcurrencyException>(() => session.SaveChangesAsync());
+            });
+            await amre.WaitAsync();
+            using (var session = stores[1].OpenAsyncSession(new SessionOptions {TransactionMode = TransactionMode.ClusterWide}))
+            {
+                var loaded = await session.LoadAsync<TestObj>(entityId);
+                loaded.Prop = "Changed";
+                await session.SaveChangesAsync();
+                amre2.Set();
+            }
+
+            await task;
+        }
+
+        [Fact]
+        public async Task ClusterWideTransaction_WhenLoadAndUpdateWhileDeleted_ShouldSucceedOnlyInTheFirst()
+        {
+            var (nodes, leader) = await CreateRaftCluster(3, shouldRunInMemory: false);
+
+            using var documentStore = GetDocumentStore(new Options {Server = leader, ReplicationFactor = nodes.Count});
+
+            var entity = new TestObj();
+            using (var session = documentStore.OpenAsyncSession(new SessionOptions { TransactionMode = TransactionMode.ClusterWide }))
+            {
+                await session.StoreAsync(entity);
+                await session.SaveChangesAsync();
+            }
+
+            await LoadAndUpdateWhileDeleted(nodes, documentStore.Database, entity.Id);
+        }
+        
+        [Fact]
+        public async Task ClusterWideTransaction_WhenImportThenLoadAndUpdateWhileDeleted_ShouldSucceedOnlyInTheFirst()
```

Comment:
Effectively the same as `ClusterWideTransaction_WhenLoadAndUpdateWhileDeleted_ShouldSucceedOnlyInTheFirst`, no?

Reply:
Maybe it is not necessary.
Just checking the same scenario after export import.
Should I remove it?